ui/public/locales/en.json

```diff
@@ -2637,22 +2637,22 @@
 "message.volume.state.attaching": "The volume is attaching to a volume from Ready state.",
 "message.volume.state.copying": "Volume is copying from image store to primary, in case it's an uploaded volume.",
 "message.volume.state.creating": "The volume is being created.",
-"message.volume.state.destroy": "The volume was destroyed, and cannot be recovered.",
+"message.volume.state.destroy": "The volume has been destroyed and cannot be recovered.",
```

Comment:
This tooltip appears when a volume is in the destroyed state. But if the volume is not expunged, it can still be recovered. I think we should remove the last part of this phrase.

Reply:
Makes perfect sense, thanks for your input. I'll change it.